tests/integration/feeds/test_feeds_client.py

```diff
@@ -8,17 +11,19 @@
 from nisystemlink.clients.core import ApiException
 from nisystemlink.clients.feeds import FeedsClient
 from nisystemlink.clients.feeds.models import CreateFeedRequest, Platform
-
+from uplink import retry
 
 FEED_DESCRIPTION = "Sample feed for uploading packages"
 PACKAGE_PATH = str(
     Path(__file__).parent.resolve()
     / "test_files"
     / "sample-measurement_0.5.0_windows_x64.nipkg"
 )
+PREFIX = "Feeds Client Test -"
 
 
 @pytest.fixture(scope="class")
+@retry(when=retry.when.status(429), stop=retry.stop.after_attempt(5))
```

Comment:
I don't think this is going to do anything, did you see it work? I thought you needed the retry annotating the client function or the client class itself. We have a PR up as an example of how to add retry to a client that I think needs more work but it would be fine to use for a test. Or we could add this retry to the feeds client class, which we've done for other clients.